web/src/ts/types/index.ts

```diff
@@ -0,0 +1,6 @@
+export interface IUser {
+  createdAt: string | null
+  email: string | null
+  id: number | null
+  username: string | null
```

Comment:
This data can't be null. If you have user, you will have data
Just in other plase you should check like `IUser || {}`
`IUser || empty object`